tests/suites/os/suite.js

```diff
@@ -100,11 +101,16 @@ module.exports = {
 	},
 	tests: [
 		'./tests/fingerprint',
+		'./tests/os-release',
+		'./tests/chrony',
+		'./tests/kernel-overlap',
+		'./tests/bluetooth',
+		'./tests/healthcheck',
+		'./tests/variables',
 		'./tests/led',
 		'./tests/config-json',
 		// The boot splash test is currently disabled because of the excessive time spent on it.
 		// './tests/boot-splash',
 		'./tests/connectivity',
-		'./tests/engine-healthcheck',
```

Comment:
@vipulgupta2048 the tests are inside their own folders, because the test may ahve other associated assets (e.g dockerfiles) with them - and this seemed like a way to keep all of those together. 

Removing the engine healthcheck is a mistake, will fix